compiler/testData/diagnostics/tests/delegation/Delegation_MultipleDelegates.txt

```diff
@@ -9,7 +9,7 @@ internal trait One {
 }
 
 internal trait OneImpl : One {
-    invisible_fake open override /*1*/ /*fake_override*/ fun boo(): kotlin.Int
+    invisible_fake final override /*1*/ /*fake_override*/ fun boo(): kotlin.Int
```

Comment:
Do you have any specific case that needs to be checked? Otherwise I'd rather trust the completeness of our test suite and commit the change.